src/Symfony/Component/HttpFoundation/composer.json

```diff
@@ -17,6 +17,7 @@
     ],
     "require": {
         "php": ">=8.2",
+        "ext-sodium": "*",
```

Comment:
Making HttpFoundation require ext-sodium is a no-go (especially if only the UriSigner needs it as it is not the main feature of the component)